### ISSUE TITLE
Support for different SSH port

### ISSUE THREAD

**Issue by rmdort** [ucbrise/clipper#90]
Can you support different sshd port when initializing clipper.

````
clipper = cm.Clipper(host, PORT_NUMBER, user, key)
````

When i specify port within the IP, redis attempts to connect to incorrect address

````
Could not connect to Redis at XXX:XX:XX:XX:2201:6379: nodename nor servname provided, or not known
````

**Comment by dcrankshaw** [comment]
Sure. Just opened #91 to support this.

**Comment by dcrankshaw** [comment]
Fixed by #91
